api/server/gin_middlewares.go

```diff
@@ -31,6 +31,11 @@ var (
 	apiResponseCountMeasure = common.MakeMeasure("api/response_count", "API response count", stats.UnitDimensionless)
 	apiLatencyMeasure       = common.MakeMeasure("api/latency", "Latency distribution of API requests", stats.UnitMilliseconds)
 
+	// ambiguously named, both of these measure responses from post function invocation (or, if the function
+	// cannot be placed, after that) - critically, only expected errors or function responses.
+	invokeResponseCountMeasure      = common.MakeMeasure("api/invoke_count", "Invoke response count", stats.UnitDimensionless)
+	httpTriggerResponseCountMeasure = common.MakeMeasure("api/http_trigger_count", "HTTP trigger response count", stats.UnitDimensionless)
```

Comment:
i'm speechless for names, have at it... describing what this is exactly is not intuitive imo either (and makes me think it's all wrong, but meh)